src/App.js

```diff
@@ -1,11 +1,39 @@
-import React from "react";
+import React, {useState} from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
-import { Button } from './Components/Button/Button.js'
+import { Container, Col, Row } from 'react-bootstrap';
+import { Navbar } from './Components/Navbar/Navbar.js';
+import { HomePage } from './Components/HomePage/HomePage.js';
+import { Banner } from './Components/Banner/Banner.js';
 
 function App() {
+  const [ page, updatePage ] = useState('Home');
+
+  const changePage = (directTo) => {
+    updatePage(directTo)
+  }; 
+
+  const renderPage = (page) => {
+    console.log('Rendered')
+    if (page === 'Home') {return <HomePage/>};
+    if (page === 'Profile') {return <div>Profile page placeholder</div>};
+    if (page === 'Details') {return <div>Details page placeholder</div>};
+    if (page === 'Log in') {return <div>Log in page placeholder</div>};
+  };
+
   return (
     <div>
-      <Button label={'Home'} type={'primary'}></Button>
+      <Container>
+        <Row>
+          <Col md='auto'><Banner/></Col>
```

Comment:
you can define width of based on number of columns it takes in the row. Auto will evenly space them which is why you need an empty column